reana_server/rest/secrets.py

```diff
@@ -112,7 +112,10 @@ def add_secrets():  # noqa
               }
```

Comment:
Originally, we have set the [`access_token` as a required parameter](https://reana-server.readthedocs.io/en/latest/_static/api.html#operation/get_workflows). This is no longer true, can you please update the docstring? This will be applicable for all endpoints which need authentication :).